autoPyTorch/api/base_task.py

```diff
@@ -229,7 +229,11 @@ def __init__(
                                  " HyperparameterSearchSpaceUpdates got {}".format(type(self.search_space_updates)))
 
     @abstractmethod
```

Comment:
It would be good to mark the class as ABC as it contains methods are now abstractmethods.

```python
class BaseTask(ABC):
    ...
```